ssl/quic/quic_record_tx.c

```diff
@@ -509,12 +509,16 @@ static int qtx_encrypt_into_txe(OSSL_QTX *qtx, struct iovec_cur *cur, TXE *txe,
         nonce[nonce_len - i - 1] ^= (unsigned char)(pn >> (i * 8));
 
     /* type and key will already have been setup; feed the IV. */
-    if (EVP_CipherInit_ex(cctx, NULL, NULL, NULL, nonce, /*enc=*/1) != 1)
+    if (EVP_CipherInit_ex(cctx, NULL, NULL, NULL, nonce, /*enc=*/1) != 1) {
+        ERR_raise(ERR_LIB_SSL, ERR_R_EVP_LIB);
```

Comment:
Again there are cases above where we simply return 0 without raising an error, but here we are.